### PR TITLE
Install explicit version of "pip"

### PR DESCRIPTION
**Pull request by tianon**

### CODE REVIEW

File: 2.7/Dockerfile [hunk 1]
```diff
@@ -12,6 +12,9 @@ RUN gpg --keyserver ha.pool.sks-keyservers.net --recv-keys C01E1CAD5EA2C4F0B8E35
 
 ENV PYTHON_VERSION 2.7.9
 
+# if this is called "PIP_VERSION", pip explodes with "ValueError: invalid truth value '<VERSION>'"
+ENV _PIP_VERSION 6.1.1
```
Comment (Starefossen) [inline 2.7/Dockerfile]: I would maybe have preferred naming it something less awkward like `DOCKER_PIP_VERSION` or `PYTHON_PIP_VERSION`.

Comment (tianon) [inline 2.7/Dockerfile]: That's definitely easy to change. :+1:

Comment (Starefossen) [inline 2.7/Dockerfile]: Otherwise this PR looks solid to me :+1:

Comment (tianon) [inline 2.7/Dockerfile]: Updated! :heart:

**Comment by yosifkit** [comment]
LGTM

**Pull request merged**